### ISSUE TITLE
failed to build Mozc.app

### ISSUE THREAD

**Issue by ivan05095** [google/mozc#430]
Ivans-MacBook-Pro:src ivan$ python build_mozc.py build -c Release mac/mac.gyp:GoogleJapaneseInput mac/mac.gyp:gen_launchd_confs
INFO: Running: ninja -C out_mac/Release GoogleJapaneseInput gen_launchd_confs
Traceback (most recent call last):
  File "build_mozc.py", line 1239, in <module>
    main()
  File "build_mozc.py", line 1226, in main
    BuildMain(cmd_opts, cmd_args)
  File "build_mozc.py", line 857, in BuildMain
    BuildWithNinja(options, targets)
  File "build_mozc.py", line 831, in BuildWithNinja
    RunOrDie([ninja, '-C', build_arg] + ninja_targets)
  File "/Users/ivan/work/mozc/src/build_tools/util.py", line 92, in RunOrDie
    process = subprocess.Popen(argv)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 1335, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory

when i am building the app for mac it shows likes these? how do i solve it? thanks

**Comment by lazmond3** [comment]
This is the same problem I also get, but for unix/ibus/... targets.

**Comment by akirakubo** [comment]
I was able to reproduce the subcommand failure on macOS 10.13.6 (High Sierra) and Xcode 10 with the latest updates.

I opened ```src/third_party/breakpad/src/client/mac/sender/Breakpad.xib``` with Xcode and changed "Builds for" setting from "Unknown macOS Version" to any one in the drop down list (I chose "macOS 10.6 and Later"). After that, build passed.

!["Builds for" setting](https://user-images.githubusercontent.com/1080143/50557280-1d6bcb00-0d27-11e9-8976-e391896f04b8.png)

As the error message says, ```src/mac/{English,Japanese}.lproj/Config.xib``` seems to have the same problem. It didn't affect passing build, but Mozc input menu items such as "Preferences..." didn't show in the menu bar unless I changed the setting.

![No input menu in menu bar](https://user-images.githubusercontent.com/1080143/50557368-f235ab80-0d27-11e9-99be-723fe0af87db.png)
